targettest/sym1/symTiny.c

```diff
@@ -0,0 +1,40 @@
+// --------------------------------------------------------------------------
+// Hello World for Sym-1
+//
+// Uses only getchar, putchar and puts, generating smaller code than printf
+//
+// Wayne Parham
+//
+// wayne@parhamdata.com
+// --------------------------------------------------------------------------
+
+#include <stdio.h>
+#include <sym1.h>
+
+void main(void) {
+   char c = 0x00;
+   int  d = 0x00;
+   int  l = 0x00;
+
+   puts( "Hello World!\n" );
+
+   puts( "Type a line and press ENTER, please:\n" );
+
+   for( l = 0; l < 2; l++ ) {
+      beep();
+      for( d = 0; d < 10 ; d++ ) {
+      }
+   }
+
+   while( c != '\n' ) {
+      c = getchar();
+   }
+
+   puts( "\n\nThanks!\n" );
+
+   for( l = 0; l < 5; l++ ) {
+      beep();
+      for( d = 0; d < 10 ; d++ ) {
+      }
+   }
+}
```

Comment:
Please adhere to the cc65 code style, see symDisplay.c.

Reply:
Updated.  Please check and confirm.

Reply:
Please adhere to the cc65 code style, see symDisplay.c.

Reply:
Fixed.  Please check and confirm.